api-spec/extensions/fields.fragment.yaml

```diff
@@ -42,3 +41,9 @@ components:
           type: array
           items:
             type: string
+      example:
+        include:
+          - id,properties.eo:cloud_cover
+        exclude:
+          - geometry,properties.datetime
+
```

Comment:
Is it implicit that we will allow wildcards? If yes then I'd include one as example.

Reply:
I wasn't sure I wanted to force wildcard in, but if others feel strongly, I'm not against the idea. I worry that once we start using "smart" fields we will need something like regex. I was hoping that explicitly naming the fields would be enough to start.